core/src/test/java/org/mskcc/cbio/oncokb/util/IndicatorUtilsTest.java

```diff
@@ -77,6 +69,48 @@ public void testProcessQuery() throws Exception {
         assertEquals("The highest resistance level should be R1",
             LevelOfEvidence.LEVEL_R1, indicatorQueryResp.getHighestResistanceLevel());
 
+
+        /**
+         * Comparing between two queries
+         */
+        Query query1, query2;
+        IndicatorQueryResp resp1, resp2;
+
+        // Match Gain with Amplification
+        query1 = new Query("PTEN", "Gain", null);
+        query2 = new Query("PTEN", "Amplification", null);
+        resp1 = IndicatorUtils.processQuery(query1, null, null, null, false);
+        resp2 = IndicatorUtils.processQuery(query2, null, null, null, false);
+        assertTrue("The oncogenicity should be the same", resp1.getOncogenic().equals(resp2.getOncogenic()));
+        assertTrue("The treatment should be the same", resp1.getTreatments().equals(resp2.getTreatments()));
+
+        // Match Loss with Deletion
+        query1 = new Query("PTEN", "Loss", null);
+        query2 = new Query("PTEN", "Deletion", null);
+        resp1 = IndicatorUtils.processQuery(query1, null, null, null, false);
+        resp2 = IndicatorUtils.processQuery(query2, null, null, null, false);
+        assertTrue("The oncogenicity should be the same", resp1.getOncogenic().equals(resp2.getOncogenic()));
```

Comment:
Could we change the message to something like "The oncogenicities are not the same"? 
I think the error message should describe what is the error precisely.  And same for the treatment message